### PR TITLE
Add PoolUtil swap direction & token type util functions

### PR DESCRIPTION
**Pull request by odcheung**
- Add util functions to PoolUtil to help identify swap direction & token A/B for token mints

### CODE REVIEW

File: sdk/src/utils/public/types.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export enum SwapDirection {
+  AtoB = "aToB",
+  BtoA = "bToA",
+}
+
+export enum TokenType {
+  TokenA = 1,
+  TokenB,
```
Comment (terranmoccasin) [inline sdk/src/utils/public/types.ts:8]: Shoudl this be `TokenB = 2,` ?

File: sdk/src/utils/public/pool-utils.ts [hunk 2]
```diff
@@ -20,6 +25,44 @@ export class PoolUtil {
...
   }
 
+  /**
+   * Return the corresponding token type (TokenA/B) for this mint key for a Whirlpool.
+   *
+   * @param pool The Whirlpool to evaluate the mint against
+   * @param mint The token mint PublicKey
+   * @returns The match result in the form of TokenType enum. undefined if the token mint is not part of the trade pair of the pool.
+   */
+  public static getTokenType(pool: WhirlpoolData, mint: PublicKey) {
```
Comment (terranmoccasin) [inline sdk/src/utils/public/pool-utils.ts:35]: nit: add method return types for `getTokenType` and `getSwapDirection`

**Review by terranmoccasin — APPROVED** [review]
lgtm - just had two small comments

File: sdk/src/utils/public/types.ts (hunk 1 above)
Comment (odcheung) [reply sdk/src/utils/public/types.ts:8]: Typescript will automatically fill that in if you specify the starting number

**Pull request merged**